magento/pwa-studio
# [PWA-178] Gift options support in cart page

## Description

Gift Options Support in Venia Cart Page

## Related Issue
Closes [PWA-178](https://jira.corp.magento.com/browse/PWA-178)

### Verification Stakeholders
@soumya-ashok 
@jimbo 

### Verification Steps
1. Add an item to the cart and go to the cart page.
2. Expand the `See Gift Options` section.
3. You should see the Gift Options.

## Screenshots / Screen Captures
![image](https://user-images.githubusercontent.com/35203638/73213859-a0bc4380-4116-11ea-9e1e-e1c64769a2e4.png)

![image](https://user-images.githubusercontent.com/35203638/73214364-a403ff00-4117-11ea-845f-2fedc2a42543.png)

## Checklist
* GQL does not support Gift Options yet. Make sure a rest call is made to save the options.
